runtime/src/lib.rs

```diff
@@ -412,15 +412,15 @@ impl pallet_treasury::Trait for Runtime {
 }
 
 parameter_types! {
-	pub const ContractTransferFee: Balance = 1 * NANOCENTS;
-	pub const ContractCreationFee: Balance = 1 * MICROCENTS;
-	pub const ContractTransactionBaseFee: Balance = 1 * NANOCENTS;
-	pub const ContractTransactionByteFee: Balance = 10 * MICROCENTS;
-	pub const ContractFee: Balance = 1 * CENTS;
-	pub const TombstoneDeposit: Balance = 1 * DOLLARS;
-	pub const RentByteFee: Balance = 1 * DOLLARS;
-	pub const RentDepositOffset: Balance = 1000 * DOLLARS;
-	pub const SurchargeReward: Balance = 150 * DOLLARS;
+	pub const ContractTransferFee: Balance = 1 * MICROS;
```

Comment:
tried to adjust these values in relative proportion to current substrate defaults.
as mentioned in description contract fees need some deeper review